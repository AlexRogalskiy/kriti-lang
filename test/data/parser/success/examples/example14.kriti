{ "foo": "x-y", "bar": "a b cde-fgh" }