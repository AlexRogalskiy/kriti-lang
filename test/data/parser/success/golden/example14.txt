Object
    ( fromList
        [
            ( "bar"
            , String "obj['a b cde-fgh']"
            )
        ,
            ( "foo"
            , String "obj['x-y']"
            )
        ]
    )